cli/kv_command.go

```diff
@@ -174,8 +180,95 @@ func (c *kvCommand) lsAction(_ *fisk.ParseContext) error {
 		return err
 	}
 
+	if c.bucket != "" {
+		err = c.lsBucketKeys(mgr)
+	} else {
+		err = c.lsBuckets(mgr)
+	}
+
+	return err
+}
+
+func (c *kvCommand) lsBucketKeys(mgr *jsm.Manager) error {
+	if mgr == nil {
+		return errors.New("mgr cannot be nil")
+	}
+
+	// Is this a KV bucket?
+	_, js, err := prepareJSHelper()
+	if err != nil {
+		return fmt.Errorf("unable to prepare js helper: %s", err)
+	}
+
+	kv, err := js.KeyValue(c.bucket)
+	if err != nil {
+		return fmt.Errorf("unable to load bucket: %s", err)
+	}
+
+	keys, err := kv.Keys()
+	if err != nil {
+		if err == nats.ErrNoKeysFound {
+			return fmt.Errorf("no keys found in bucket %s", c.bucket)
```

Comment:
I dont think this is an error condition, see for example how we handle no streams, we just print a message and clean exit

https://github.com/nats-io/natscli/blob/7b15a8de2c742fac2083759285b59c098ae49ea7/cli/stream_command.go#L2416

Reply:
Sounds good. Will update.